repository/kafka/README.md

```diff
@@ -0,0 +1,9 @@
+# Kafka Operator
+
+The KUDO Kafka operator creates, configures and manages [Apache Kafka](https://kafka.apache.org/) clusters running on Kubernetes
+
+## Getting started
+
+The latest stable version of Kafka operator is `0.1.1`
```

Comment:
do you really want to publish docs for every bugfix release? wouldn't that be too much work? Maybe `0.1` folder would be enough since everything 0.1.X should be just bugfixes? But up yo you :)

Reply:
yeah right! I don't have a decision or any reasons for making it 0`0.1.1` so will make it `0.1`
thanks!